packages/react-component-library/src/components/RangeSlider/Tick.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import { SliderItem } from 'react-compound-slider'
+
+interface TickProps {
+  tick: SliderItem
+  count: number
+  hasLabels?: boolean
+  values: ReadonlyArray<number>
+  domain: ReadonlyArray<number>
+  reversed?: boolean
+}
+
+function isActive(values: ReadonlyArray<number>, tickValue: number): boolean {
+  const valuesBelow = values.some(item => item <= tickValue)
+  const valuesAbove = values.some(item => item >= tickValue)
+
+  if (
+    (values.length === 1 && values[0] >= tickValue) ||
```

Comment:
Wondering if some variable names and then a ternary operator could work well here.

Reply:
Good shout - this has now been simplified